doc/_templates/class.rst

```diff
@@ -9,7 +9,6 @@
 
 .. autoclass:: {{ objname }}
    :inherited-members:
-   :undoc-members:
```

Comment:
I don't think that this should be removed. If someone contributes a class and forgets to add a docstring to a method (which could happen if we are not rigorous enough with our reviews) the resulting method will no longer be documented